docs/UserGuide.md

```diff
@@ -549,9 +740,9 @@ Moves student from one group to another group.
 #### Output if command fails
 
 - Displays message indicating either:
-  - [Invalid command format](#mistake-1-invalid-command-format)
+  - [Invalid command format.](#mistake-1-invalid-command-format)
   - [No such student to move.](#mistake-4-no-such-student-to-move)
-  - Invalid destination path.
+  - [Invalid destination path.](#mistake-2--invalid-path)
```

Comment:
This link is invalid. Nothing happens after clicking on it. Perhaps the link contains a redundant '-', which causes it to be invalid.